ddtrace/contrib/django/apps.py

```diff
@@ -47,19 +47,24 @@ def ready(self):
         )
 
         if settings.AUTO_INSTRUMENT:
-            # trace Django internals
-            insert_exception_middleware()
-            try:
-                patch_db(tracer)
-            except Exception:
-                log.exception('error patching Django database connections')
+            if settings.AUTO_INSTRUMENT_TEMPLATE.get('EXCEPTION_MIDDLEWARE', True):
+                # trace Django internals
+                insert_exception_middleware()
 
-            try:
-                patch_template(tracer)
-            except Exception:
-                log.exception('error patching Django template rendering')
+            if settings.AUTO_INSTRUMENT_TEMPLATE.get('TEMPLATES', True):
```

Comment:
we can remove the `AUTO_INSTRUMENT_TEMPLATE` and just use `INSTRUMENT_TEMPLATE` for Django templates